#!/bin/sh /usr/share/dpatch/dpatch-run
## 15-makefiles.dpatch by Michael Meskes <meskes@debian.org>
##
## DP: Add missing makefiles.

@DPATCH@

--- virtualbox-ose-1.6.4-dfsg/src/apps/Makefile.kmk	1970-01-01 01:00:00.000000000 +0100
+++ Makefile.kmk	2008-08-09 21:40:09.000000000 +0200
@@ -0,0 +1,36 @@
+# $Id: Makefile.kmk 34052 2008-08-04 17:27:28Z klaus $
+## @file
+# Top-level makefile for the external apps.
+#
+
+#
+# Copyright (C) 2006-2007 Sun Microsystems, Inc.
+#
+# This file is part of VirtualBox Open Source Edition (OSE), as
+# available from http://www.virtualbox.org. This file is free software;
+# you can redistribute it and/or modify it under the terms of the GNU
+# General Public License (GPL) as published by the Free Software
+# Foundation, in version 2 as it comes in the "COPYING" file of the
+# VirtualBox OSE distribution. VirtualBox OSE is distributed in the
+# hope that it will be useful, but WITHOUT ANY WARRANTY of any kind.
+#
+# Please contact Sun Microsystems, Inc., 4150 Network Circle, Santa
+# Clara, CA 95054 USA or visit http://www.sun.com if you need
+# additional information or have any questions.
+#
+
+DEPTH = ../..
+include $(KBUILD_PATH)/header.kmk
+
+ifdef VBOX_WITH_KCHMVIEWER
+ifneq ($(wildcard kchmviewer-2.6),)
+SUBDIRS.linux = kchmviewer-2.6
+endif
+endif
+
+ifneq ($(wildcard tunctl),)
+SUBDIRS.linux += tunctl
+endif
+
+include $(KBUILD_PATH)/rules.kmk
+
--- virtualbox-ose-1.6.4-dfsg/src/apps/tunctl/Makefile.kmk	1970-01-01 01:00:00.000000000 +0100
+++ Makefile.kmk	2008-08-09 21:42:10.000000000 +0200
@@ -0,0 +1,35 @@
+# $Id: Makefile.kmk 31342 2008-05-28 08:44:35Z frank $
+## @file
+# Makefile for VBoxTunctl, which is identical to tunctl from Usermode Linux
+# except for the name.
+#
+
+#
+# Copyright (C) 2006-2007 Sun Microsystems, Inc.
+#
+# This file is part of VirtualBox Open Source Edition (OSE), as
+# available from http://www.virtualbox.org. This file is free software;
+# you can redistribute it and/or modify it under the terms of the GNU
+# General Public License (GPL) as published by the Free Software
+# Foundation, in version 2 as it comes in the "COPYING" file of the
+# VirtualBox OSE distribution. VirtualBox OSE is distributed in the
+# hope that it will be useful, but WITHOUT ANY WARRANTY of any kind.
+#
+# Please contact Sun Microsystems, Inc., 4150 Network Circle, Santa
+# Clara, CA 95054 USA or visit http://www.sun.com if you need
+# additional information or have any questions.
+#
+
+DEPTH = ../../..
+include $(KBUILD_PATH)/header.kmk
+
+ifneq ($(KBUILD_HOST),linux)
+$(error "VBoxTunctl can only be built on Linux!")
+endif
+
+PROGRAMS       = VBoxTunctl
+
+VBoxTunctl_TEMPLATE = VBOXR3EXE
+VBoxTunctl_SOURCES  = tunctl.c
+
+include $(KBUILD_PATH)/footer.kmk
